src/App/Pages/Send/SendGroupingsPage/SendGroupingsPageListItem.cs

```diff
@@ -66,5 +67,31 @@ public string Icon
                 return _icon;
             }
         }
+
+        public string AutomationId
+        {
+            get
+            {
+                if (_name != null)
+                {
+                    _automationId = "SendItem";
+                }
+                if (Type != null)
+                {
+                    switch (Type.Value)
+                    {
+                        case SendType.Text:
+                            _automationId = "SendTextFilter";
+                            break;
+                        case SendType.File:
+                            _automationId = "SendFileFilter";
+                            break;
+                        default:
+                            break;
+                    }
+                }
+                return _automationId;
+            }
+        }
```

Comment:
⛏️  Right now `_automationId` is not useful. If the `_automationId` is meant to be used as a cache you should add something like this at beginning:
```c#
if (_automationId != null)
{
    return _automationId;
}
```
However, as far as I can see this property won't be triggered more than once per item so you can directly return the value on each part to avoid the field:
```c#
if (_name != null)
{
    return "SendItem";
}
if (Type != null)
{
    switch (Type.Value)
    {
        case SendType.Text:
            return  "SendTextFilter";
        case SendType.File:
            return  "SendFileFilter";
    }
}
return null;
```